english/order-pizza/order-pizza.md

```diff
@@ -1,3 +1,10 @@
-Your hungry and tired, pizza is the only thing that will cheer you up.
```

Comment:
Lines where not deleted here just modified, apparently the robot isn't so smart.  No reason this could not be merged.